docs/release-checklist.md

```diff
@@ -9,7 +9,7 @@
         1. Follow instructions to download the nuget.exe from [here](https://learn.microsoft.com/en-us/nuget/install-nuget-client-tools)
         1. Run `./scripts/UploadPackages.ps1 -PackageDirectory <downloads>/drop_build_main -NuGetPath <nuget_tool_directory>`
         1. You need to be part of the armdeployments org on nuget.org. (Ask one of the admins to be added) You must generate an API key and then use that as the password for when the popup window appears after running the above command. (Username can be anything)
-    1. Bump the Bicep.Types NuGet package version in this project in this [file](https://github.com/Azure/bicep/blob/main/src/Bicep.Core/Bicep.Core.csproj) by creating and merging a PR
+    1. Bump the Bicep.Types.Az NuGet package version in this project in this [file](https://github.com/Azure/bicep/blob/main/src/Bicep.Core/Bicep.Core.csproj) by creating and merging a PR
```

Comment:
Fixed line 7

Reply:
Also do you mind updating line 19 - 20 to

```
1. Get version number from official build by looking at the artifacts and push a new tag to the Bicep repo. This should be of format `vXX.YY.ZZ` - e.g `v0.14.85`.
    1. In the Bicep repo, make sure you are on the main branch and on the commit that matches the submodule commit that triggers the official build in the BicepMirror repository.
        - This can be done by running `git reset <submodule_commit_id> --hard` in your local Bicep repository folder.
```?